Gruntfile.js

```diff
@@ -33,7 +33,7 @@ module.exports = function(grunt) {
 
         babel: {
           options: {
-            sourceMap: true,
+            sourceMaps: false,
```

Comment:
@crisner Also the sourceMaps - do we need to have the .map files available? The tests don't require them. @sagarpreet-chadha

Reply:
From what I understand sourceMaps help in debugging, provides line numbers to narrow down where the issue is from.

Reply:
Ok, I can add that back in easy enough.